packages/esm-ward-app/src/ward-patient-card/bento-elements/ward-patient-name.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { type WardPatientCardBentoElement } from '../../types';
+import styles from '../ward-patient-card.scss';
+
+const WardPatientName: WardPatientCardBentoElement = ({ patient }) => {
+  // TODO: make server return patient.display and use that instead
+  const { givenName, familyName } = patient?.person?.preferredName;
```

Comment:
Better to use `patient?.person?.display` (`patient?.display` has the primary identifier as well).

Reply:
Also there is likely a utility method to get a patient name (or should be) for display.  There may be active work ongoing around this, but the way a patient name is formatted is configurable on the backend, and should likely be respected.

Reply:
There is a `displayName` function from esm-framework, but it takes in a FHIR patient instead of an OpenMRS Patient. I'll use `patient?.person?.display` for now.

Reply:
Actually... the REST endpoint we are using (`/adminLocation`) does not return `patient.person.display` even with `v=full`. I'll keep this (and the TODO) for now.

Reply:
`patient?.person?.display`, when it exists, is already formatted according to the backend name template. We sort of need a utility function for FHIR, because the display name is (in FHIRPath syntax) `Patient.name.where(use='usual').text.first()`, but we have a fallback if the `text` element is not defined which happens in cases where we've constructed a pseudo-FHIR patient out of the results of the REST API.